cmd/context/delete.go

```diff
@@ -40,7 +40,7 @@ func DeleteCMD() *cobra.Command {
 	return cmd
 }
 
-func Delete(ctx context.Context, okCtx string) error {
+func Delete(_ context.Context, okCtx string) error {
```

Comment:
Should we completely remove if we are not using the context on this?

`func Delete(okCtx string) error `

Reply:
Agree, if the param is not being used and the function doesn't have to satisfy any interface, we should remove it then

Reply:
i was conservative in this sense, as this removing would lead to any other changes, i will add a commit with this removals so the review is clear